data/Learning_Pathway_Index.csv

```diff
@@ -1,1398 +1,1397 @@
-Data Engineer - Google Cloud Big Data and Machine Learning Fundamentals,Google Cloud Skill Boost,Beginner,Free during mentorship period,Data Engineering for Streaming Data: Implementing streaming pipelines on Cloud Dataflow,3 minutes,Medium,"data engineer, streaming data, pipelines, cloud dataflow, Google Cloud, Big data, Machine Learning",https://www.cloudskillsboost.google/course_sessions/5007143/video/383343
-Data Engineer - Google Cloud Big Data and Machine Learning Fundamentals,Google Cloud Skill Boost,Beginner,Free during mentorship period,Data Engineering for Streaming Data: Visualization with Looker,2 minutes,Medium,"data engineer, streaming data, visualization, Looker, Google Cloud, Big data, Machine Learning",https://www.cloudskillsboost.google/course_sessions/5007143/video/383344
-Data Engineer - Google Cloud Big Data and Machine Learning Fundamentals,Google Cloud Skill Boost,Beginner,Free during mentorship period,Data Engineering for Streaming Data: Visualization with Data Studio,1 minute,Medium,"data engineer, streaming data, visualization, Data Studio, Google Cloud, Big data, Machine Learning",https://www.cloudskillsboost.google/course_sessions/5007143/video/383345
-Data Engineer - Google Cloud Big Data and Machine Learning Fundamentals,Google Cloud Skill Boost,Beginner,Free during mentorship period,Data Engineering for Streaming Data: Lab introduction: Creating a streaming data pipeline for a Real-Time dashboard with Dataflow,1 minute,Medium,"data engineer, streaming data, lab, pipeline, Dataflow, Google Cloud, Big data, Machine Learning",https://www.cloudskillsboost.google/course_sessions/5007143/video/383346
-Data Engineer - Google Cloud Big Data and Machine Learning Fundamentals,Google Cloud Skill Boost,Beginner,Free during mentorship period,Data Engineering for Streaming Data: Creating a Streaming Data Pipeline for a Real-Time Dashboard with Dataflow,60 minutes,Medium,"data engineer, streaming data, pipeline, Dataflow, Google Cloud, Big data, Machine Learning",https://www.cloudskillsboost.google/course_sessions/5007143/labs/383347
-Data Engineer - Google Cloud Big Data and Machine Learning Fundamentals,Google Cloud Skill Boost,Beginner,Free during mentorship period,Data Engineering for Streaming Data: Summary,1 minute,Medium,"data engineer, streaming data, Google Cloud, Big data, Machine Learning",https://www.cloudskillsboost.google/course_sessions/5007143/video/383348
-Data Engineer - Google Cloud Big Data and Machine Learning Fundamentals,Google Cloud Skill Boost,Beginner,Free during mentorship period,Data Engineering for Streaming Data: Quiz,2 minutes,Medium,"data engineer, streaming data, quiz, Google Cloud, Big data, Machine Learning",https://www.cloudskillsboost.google/course_sessions/5007143/quizzes/383349
+Course_Learning_Material,Source,Course_Level,Type,Module,Difficulty_Level,Keywords,Links,Duration_in_minutes
```

Comment:
We do not need a `Duration_in_minutes` field the Duration should be human readable, for EDA purposes you can create a runtime/temp field if you like to do some stats on it - pls re-introduce the original Duration field and corresponding data

Reply:
`Duration` isnt a numeric field, its a string and also can be expresses in minutes, hours or days where applicable.